src/reset.css

```diff
@@ -0,0 +1,14 @@
+h2,
+h1,
+p {
+  padding: 0;
+  margin: 0;
+  font-size: inherit;
+  font-family: inherit;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif ;
+}
```

Comment:
```suggestion
body {
  margin: 0;
  box-sizing: border-box;
  font-family: Roboto, sans-serif ;
}
```
fonts should not be in the reset file

Reply:
Agree with @ShmelyovOleg , you have it on normalise file

Reply:
Right, don’t need it anymore 
<img width="427" alt="Screenshot 2022-07-20 at 13 19 07" src="https://user-images.githubusercontent.com/47606379/179958937-3707260a-d516-44cf-a5de-9edb531c10b0.png">